app/upgrades/v11/upgrades.go

```diff
@@ -19,6 +19,29 @@ import (
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
+var (
+	// create ICS27 Controller submodule params, with the controller module NOT enabled
+	controllerParams = icacontrollertypes.Params{}
+
+	// create ICS27 Host submodule params with a list of allowed messages
+	hostParams = icahosttypes.Params{
+		HostEnabled: true,
+		AllowMessages: []string{
+			sdk.MsgTypeURL(&banktypes.MsgSend{}),
+			sdk.MsgTypeURL(&banktypes.MsgMultiSend{}),
+			sdk.MsgTypeURL(&distrtypes.MsgSetWithdrawAddress{}),
+			sdk.MsgTypeURL(&distrtypes.MsgWithdrawDelegatorReward{}),
+			sdk.MsgTypeURL(&govtypes.MsgVote{}),
+			sdk.MsgTypeURL(&govtypes.MsgVoteWeighted{}),
+			sdk.MsgTypeURL(&stakingtypes.MsgDelegate{}),
+			sdk.MsgTypeURL(&stakingtypes.MsgUndelegate{}),
+			sdk.MsgTypeURL(&stakingtypes.MsgCancelUnbondingDelegation{}),
+			sdk.MsgTypeURL(&stakingtypes.MsgBeginRedelegate{}),
+			sdk.MsgTypeURL(&transfertypes.MsgTransfer{}),
```

Comment:
can add authz types?


```go
sdk.MsgTypeURL(&authz.MsgExec{}),
sdk.MsgTypeURL(&authz.MsgGrant{}),
sdk.MsgTypeURL(&authz.MsgRevoke{}),
```

e.g: 
- ICA can send `authz.MsgExec(bank.Transfer(From: user, To: ica-host-addr))`  - similar to ERC20 approvals flow
- ICA can use `authz.MsgGrant` to allow grantee to restake the rewards.

Reply:
is that needed for Stride or Quicksilver?